stdlib/xmlrpc/client.pyi

```diff
@@ -15,7 +15,19 @@ class _SupportsTimeTuple(Protocol):
 
 _DateTimeComparable: TypeAlias = DateTime | datetime | str | _SupportsTimeTuple
 _Marshallable: TypeAlias = (
```

Comment:
```suggestion
# Ideally this would be a recursive type alias, but dict/list invariance makes that impractical
_Marshallable: TypeAlias = (
```

Reply:
That's not quite right because the alias *is* recursive (what is "fully recursive"?), so I put a more specific comment next to list and dict.

Reply:
Yeah, your wording is better!